server/routes/recipeRoutes.js

```diff
@@ -0,0 +1,59 @@
+// initialize the express app
+const express = require('express');
+const router = express.Router();
+
+const {
+	viewAllRecipe,
+	searchByName,
+	selectRecipe,
+} = require('../controllers/recipeController')
+
+// get all recipes
+router.get('/getAll', viewAllRecipe);
```

Comment:
should have `const {authenticate} = require('../middleware/authMiddleware');`
and fix this to `router.get('/getAll', authenticate, viewAllRecipe);` to get user object

Reply:
Resolved.